### PR TITLE
Updated Simplifed Chinese translation

### PR DESCRIPTION
**Pull request by xbmanh**
**Describe the pull request**

Provide a description of the problem your pull request solves. Screenshots, code examples, etc are welcome in this section.

**Link to relevant issues**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -99,13 +99,15 @@
...
     <string name="pref_proot_debugging_level_summary">数值越大，越冗余越高.</string>
     <string name="alert_clear_support_files_title">真的要删除支持文件吗?</string>
     <string name="alert_clear_support_files_message">
         点击“是的”将会清除支持的文件。但这不会发生在安装的应用程序的文件的。
         点击“是的”将会导致在下次运行会话或应用程序时再下载资源。
     </string>
     <string name="alert_clear_support_files_clear_button">清除</string>
-
+    <string name="review_ask_for_rating">可以在 Play商店 上为我们评分吗？</string>
+    <string name="review_ask_for_feedback">您介意在 GitHub 上反馈我们吗？</string>
```
Comment (AndrewScibek) [inline app/src/main/res/values-zh-rCN/strings.xml]: I think `您介意在 GitHub 上给我们反馈吗?` is better

**Comment by AndrewScibek** [comment]
Other then the one change the rest are good

**Comment by corbinlc** [comment]
@xbmanh will you make the change suggested by @AndrewScibek ?  It has also been a while, so please check if there are any new strings that need translation.

**Comment by MatthewTighe** [comment]
I am closing this as it seems the requested changes will not be made.

**Pull request closed without merging**